cluster/src/main/java/com/hartwig/pipeline/calling/structural/gridss/Gridss.java

```diff
@@ -35,20 +33,21 @@
 import com.hartwig.pipeline.reruns.PersistedDataset;
 import com.hartwig.pipeline.reruns.PersistedLocations;
 import com.hartwig.pipeline.resource.ResourceFiles;
-import com.hartwig.pipeline.stages.Stage;
 import com.hartwig.pipeline.stages.SubStageInputOutput;
 import com.hartwig.pipeline.storage.GoogleStorageLocation;
 import com.hartwig.pipeline.storage.RuntimeBucket;
 import com.hartwig.pipeline.tertiary.TertiaryStage;
 
-public class StructuralCaller extends TertiaryStage<StructuralCallerOutput> {
+import org.jetbrains.annotations.NotNull;
```

Comment:
Not trying to be controversial, what is the benefit of adding this annotation beyond dev-time static analysis from within Intellij?

Reply:
accidental addition, i'll remove.